app/src/main/java/de/upb/hip/mobile/activities/ExhibitDetailsActivity.java

```diff
@@ -348,14 +375,14 @@ public void displayCurrentExhibitPage() {
             bottomSheet.setVisibility(View.GONE);
         }
 
-        // display audio action only if the page provides audio
-        if (page.getAudio() == null)
+        // display audio action only if page provides audio
+        if (page.getAudio() == null) {
             displayAudioAction(false);
+        }
         else {
             displayAudioAction(true);
-
-            // TODO: continue with handling the audio
         }
+        // TODO: handle audio
```

Comment:
This TODO is only relevant if audio should start playing automatically when the page is displayed.So either this can be removed or should be at the end of the above else statement (where it was previously: "// TODO: continue with handling the audio").